docs/installation/development.rst

```diff
@@ -103,6 +92,16 @@ Create a new Plone instance, under `Add-ons`, choose the package
 Building the Documentation
 --------------------------
 
+To build the documentation you need to have the `Enchant <http://www.abisource.com/projects/enchant/>`_ library installed on your system.
+To install Enchant library on Ubuntu::
```

Comment:
Both this and the sentence below should read start "To install the Enchant library".